gnovm/tests/files/var19.gno

```diff
@@ -0,0 +1,11 @@
+package main
+
+func main() {
+	var a, b, c = 1, a+1
+	println(a)
+	println(b)
+	println(c)
+}
+
+// Error:
+// main/files/var19.gno:3: constant definition loop with a
```

Comment:
as the issue says: "xxx the error message indicates that it should be checked beforehand, possibly within predefineFileSet, and then halted."

Reply:
thx for your review. Yes my bad !
Done here: https://github.com/gnolang/gno/pull/2206/commits/bd4b92444e2494f21ab1c74c8d6b697d0ad6c755